namer/core/src/test/scala/io/buoyant/HostPortTest.scala

```diff
@@ -0,0 +1,30 @@
+package io.buoyant
+
+import com.twitter.finagle._
+import io.buoyant.test.FunSuite
+
+class HostPortTest extends FunSuite {
```

Comment:
IPv6 isn't going to work... host:port notation for ipv6 is e.g. `[2001:db8:cafe::17]:8080` and braces are not legal in Paths...